packages/perseus/src/widgets/interactive-graphs/math/angles.ts

```diff
@@ -20,19 +23,55 @@ export function polar(r: number | vec.Vector2, th: number): vec.Vector2 {
     th = (th * Math.PI) / 180;
     return [r[0] * Math.cos(th), r[1] * Math.sin(th)];
 }
-
-export const findAngle = (
-    point1: vec.Vector2,
-    point2: vec.Vector2,
-    vertex?: vec.Vector2,
+// This function calculates the angle between two points and an optional vertex.
+// If the vertex is not provided, the angle is measured between the two points.
+// This does not account for reflex angles or clockwise position.
+export const findAngleFromVertex = (
+    point: vec.Vector2,
+    vertex: vec.Vector2,
 ): number => {
-    if (vertex === undefined) {
-        const x = point1[0] - point2[0];
-        const y = point1[1] - point2[1];
-        if (!x && !y) {
-            return 0;
-        }
-        return (180 + (Math.atan2(-y, -x) * 180) / Math.PI + 360) % 360;
+    const x = point[0] - vertex[0];
+    const y = point[1] - vertex[1];
+    if (!x && !y) {
+        return 0;
     }
-    return findAngle(point1, vertex) - findAngle(point2, vertex);
+    return (180 + (Math.atan2(-y, -x) * 180) / Math.PI + 360) % 360;
+};
+
+// This function calculates the clockwise angle between three points,
+// and is used to generate the labels and equation strings of the
+// current angle for the interactive graph.
+export const getClockwiseAngle = (
```

Comment:
I should note that this is also used in the validator now, in order to preemptively solve issues with calculating the angles of reflexive graphs.